xmodule/modulestore/mixed.py

```diff
@@ -12,8 +12,15 @@
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 from opaque_keys.edx.locator import LibraryLocator
-from openedx_events.content_authoring.data import XBlockData
-from openedx_events.content_authoring.signals import XBLOCK_DELETED, XBLOCK_PUBLISHED
+from openedx_events.content_authoring.data import CourseData, XBlockData
+from openedx_events.content_authoring.signals import (
+    COURSE_CREATED,
+    # COURSE_UPDATED,
```

Comment:
I think we can use the `COURSE_PUBLISHED` event to trigger tagging updates, since there are so many places where a course's metadata gets updated.